community/examples/hpc-slinky.yaml

```diff
@@ -0,0 +1,183 @@
+      master_authorized_networks:
+      - display_name: deployment-machine
+        cidr_block: $(vars.authorized_cidr)
+      system_node_pool_enabled: false
+      configure_workload_identity_sa: true
+    outputs: [instructions]
+
+  - id: base_pool
+    source: modules/compute/gke-node-pool
+    use: [gke_cluster, node_pool_service_account]
+    settings:
+      initial_node_count: 1
+      disk_type: pd-balanced
+      machine_type: e2-standard-4
+      zones: $(vars.zones)
+
+  - id: h3_pool
+    source: modules/compute/gke-node-pool
+    use: [gke_cluster, node_pool_service_account]
+    settings:
+      initial_node_count: 0
+      disk_type: pd-balanced
+      machine_type: h3-standard-88
+      zones: $(vars.zones)
+
+  - id: slinky
+    source: community/modules/scheduler/slinky
+    use: [
+      gke_cluster,
+      # Optionally specify nodepool(s) to avoid operator components running on HPC hardware
+      base_pool
+    ]
+    settings:
+      slurm_values:
+        compute:
+          nodesets:
+          - name: debug
+            enabled: true
+            replicas: 3
+            image:
+              # Use the default nodeset image
+              repository: ""
+              tag: ""
+            resources:
+              requests:
+                cpu: 1
+                memory: 4Gi
+              limits:
+                cpu: 1
+                memory: 4Gi
+            affinity:
+              nodeAffinity:
+                requiredDuringSchedulingIgnoredDuringExecution:
+                  nodeSelectorTerms:
+                  - matchExpressions:
+                    - key: "node.kubernetes.io/instance-type"
+                      operator: In
+                      values:
+                      - e2-standard-4
+            partition:
+              enabled: true
+          - name: h3
+            enabled: true
+            replicas: 0
+            image:
+              # Use the default nodeset image
+              repository: ""
+              tag: ""
+            resources:
+              requests:
+                cpu: 86
+                memory: 324Gi
+              limits:
+                cpu: 86
+                memory: 324Gi
+            affinity:
+              nodeAffinity:
+                requiredDuringSchedulingIgnoredDuringExecution:
+                  nodeSelectorTerms:
+                  - matchExpressions:
+                    - key: "node.kubernetes.io/instance-type"
+                      operator: In
+                      values:
+                      - h3-standard-88
+            partition:
+              enabled: true
+        slurm-exporter:
+          exporter:
+            image:
+              repository: ghcr.io/slinkyproject/slurm-exporter
+              tag: "0.2.0"
+    outputs: [instructions]
+
+  - id: slurm_exporter_monitoring
+    source: modules/management/kubectl-apply
+    use: [gke_cluster]
+    settings:
+      apply_manifests:
+      - content: |
```

Comment:
We could refactor this to a separate file like we have done in [other GKE blueprints](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/057bfe21bc93e9e6fd13ed4fd37a8130000896bb/examples/gke-a4/gke-a4.yaml#L245)